unittests/FEXLinuxTests/Known_Failures

```diff
@@ -19,3 +19,5 @@ sigtest_samask.32
 sigtest_samask.64
 sigtest_sigmask.32
 sigtest_sigmask.64
+
+syscalls_efault.32
```

Comment:
Needs a comment indicating that the corresponding emulation code is intentionally not enabled currently.

Reply:
Added comment.

Reply:
As mentioned in the other discussion, "EFAULT" is misleading here. Nothing in our current handling involves EFAULT, which makes linking this comment back to the actual piece of FEX code impossible.

A better wording could be "Disabled since FEX's FaultSafeMemcpy is intentionally stub-implemented".